easybuild/easyconfigs/t/Trinity/Trinity-2.6.5-foss-2017a.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'ConfigureMake'
```

Comment:
@ebirn (cc @timeu) Can you clarify why the existing `Trinity` easyblock is not used here?

Seems to work for Trinity 2.6.6, see #6160 which was merged recently

Reply:
I cannot. I can only say that this is probably due to the authors unawareness of that specific easyblock. 
Which really would have made this whole thing much easier.

I'm testing this now, modelling it more like the #6160

Reply:
:+1: